src/forms/subscriptions.ts

```diff
@@ -274,20 +274,15 @@ class FormFields extends BaseFormFields {
     // addErrorMessageField adds a text field with a message when a trigger has
     // conditions not supported by the app
     addErrorMessageField(link: string): void {
-        let text = 'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription';
-        text += '\n\n';
-        text += makeBulletedList('Unsupported Fields', this.unsupportedFields);
-        text += '\n\n';
-        text += makeBulletedList('Unsupported Field Operators', this.unsupportedOperators);
-        text += '\n\n' + link;
+        let md = 'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription\n';
+        md += makeBulletedList('Unsupported Fields', this.unsupportedFields) + '\n';
+        md += makeBulletedList('Unsupported Field Operators', this.unsupportedOperators) + '\n';
+        md += '##### Trigger Link\n' + link;
```

Comment:
I'm curious what you think about this strategy of using an array and `.join('\n')` when we're creating multiline markdown like this

```suggestion
        const md = [
            'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription',
            makeBulletedList('Unsupported Fields', this.unsupportedFields),
            makeBulletedList('Unsupported Field Operators', this.unsupportedOperators),
            '##### Trigger Link',
            link,
        ].join('\n');
```

Reply:
Yes, I like that and used this `<array>.join()` format in other places of the code base. Thanks for the suggestion.

Reply:
It may even be a good idea to create a `markdown` function like:

```ts
export const joinMarkdown = (...lines: string[]) => {
    return lines.join('\n');
}
```

This would allow you to create multi-line markdown like this:

```ts
const md = joinMarkdown(
    'The following condition fields are not currently supported by the app. Please visit the trigger link to modify the conditions for this subscription',
    makeBulletedList('Unsupported Fields', this.unsupportedFields),
    makeBulletedList('Unsupported Field Operators', this.unsupportedOperators),
    '##### Trigger Link',
    link,
);
```

Not sure if that improves the code with that name though. There's probably a better name than `joinMarkdown`

Reply:
I did add the two small functions. `textLine` and `JoinLines`.  I'm sure there are MD libraries that would also help alleviate these steps. :)